examples/examples-cpu/nyc-taxi/hyperparameter-dask.ipynb

```diff
@@ -141,7 +139,7 @@
     "    df['tip_fraction'] = df.tip_amount / df.fare_amount\n",
     "    \n",
     "    df['pickup_weekday'] = df.tpep_pickup_datetime.dt.weekday\n",
```

Comment:
I think this one needs to be changed as well
```suggestion
    "    df['pickup_weekday'] = df.tpep_pickup_datetime.dt.isocalendar().day\n",
```

Reply:
oh, `weekday` hasn't been deprecated. Only `.weekofyear` and `.week` are mentioned in the warning.

> FutureWarning: Series.dt.weekofyear and Series.dt.week have been deprecated.  Please use Series.dt.isocalendar().week instead.

When I run the `nyc-taxi/xgboost.ipynb` notebook, for example, no warnings are generated even though it uses `.dt.weekday`.

Reply:
You are 100% right. Good lord I am not doing a good job reading that warning at all :facepalm:

Reply:
haha it's ok.

This is why I prefer `df["column_name"]` to `df.column_name`  in `pandas` code, but I don't think starting that argument is a good use of our time right now :joy: